datalake/constructs/bundled_lambda_function.py

```diff
@@ -33,6 +34,7 @@ def __init__(
             handler=f"processing.{directory}.task.lambda_handler",
             runtime=aws_lambda.Runtime.PYTHON_3_8,
             environment=environment,
+            timeout=Duration.seconds(60),
```

Comment:
We should probably add a comment to explain why not the default 3s and why not the max 15 minutes. Since this pattern is used in more than one place, it might make sense to document this in CODING.md?